# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContext.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/gl/GLContext.cpp
--- a/gfx/gl/GLContext.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/gl/GLContext.cpp	Fri May 24 11:32:54 2019 +0000
@@ -34,7 +34,7 @@
 #include "SharedSurfaceGL.h"
 #include "GfxTexturesReporter.h"
 #include "gfx2DGlue.h"
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/IntegerPrintfMacros.h"
 #include "mozilla/gfx/Logging.h"
 
@@ -276,7 +276,7 @@
       mDebugFlags(ChooseDebugFlags(flags)),
       mSharedContext(sharedContext),
       mCaps(caps),
-      mWorkAroundDriverBugs(gfxPrefs::WorkAroundDriverBugs()) {
+      mWorkAroundDriverBugs(StaticPrefs::WorkAroundDriverBugs()) {
   mOwningThreadId = PlatformThread::CurrentId();
   MOZ_ALWAYS_TRUE(sCurrentContext.init());
   sCurrentContext.set(0);